src/__swaps__/utils/swaps.ts

```diff
@@ -149,6 +149,9 @@ export const countDecimalPlaces = (number: number | string): number => {
 
 export const findNiceIncrement = (availableBalance: string | number) => {
   'worklet';
+  if (Number(availableBalance) === 0) {
+    return 0;
+  }
```

Comment:
this might not be needed with the NaN check I added, but does this hurt anything to have here?